packages/x-components/tests/unit/banner.spec.ts

```diff
@@ -60,13 +33,8 @@ describe('testing Banner component', () => {
   });
 
   it('banner is not rendered if image load fails', () => {
-    const { getBanner, getBannerImage } = mountBanner({
-      banner: createBannerStub('banner', {
-        title: 'Search UIs',
-        url: 'https://empathy.co',
-        position: 1
-      })
-    });
+    const { getBanner, getBannerImage } = render(createBannerStub('banner', bannerBase));
```

Comment:
```suggestion
    const { getBanner, getBannerImage } = render();
```

We could make `createBannerStub('banner', bannerBase)` the default value for the `banner` arg of `render`. WDYT? I know it is not a big deal here, but in any case, I think it is a good pattern to follow.``

Reply:
I like it! Reformatted for default param value